src/data/faq_de.json

```diff
@@ -2210,47 +2294,214 @@
+                    "description": "Digitale Bestätigung, die durch das Einlesen von <a href='#glossary_vaccination_certificate'>Impfzertifikaten</a> in der App erstellt wird. Sie gibt Auskunft über den gebotenen Impfschutz. Der Impfnachweis dokumentiert einen Impfvorgang mit einem vom Paul-Ehrlich-Institut empfohlenen Impfstoff. Ein Impfnachweis muss mindestens folgende Attribute enthalten: <ul><li>die personenbezogenen Daten des Geimpften (mindestens Name, Vorname und Geburtsdatum)</li><li>Datum der Schutzimpfung, Anzahl der Schutzimpfungen</li><li>Bezeichnung des Impfstoffes</li><li>Name der Krankheit, gegen die geimpft wurde sowie</li><li>Merkmale, die auf die für die Durchführung der Schutzimpfung oder die Ausstellung des <a href='#glossary_certificate'>Zertifikats</a> verantwortliche Person oder Institution schließen lassen, zum Beispiel ein offizielles Symbol oder der Name des Ausstellers.</li></ul> Für die Einreise muss nach § 2 Nummer 10 Coronavirus-Einreiseverordnung der Impfnachweis noch zusätzliche Anforderungen (bzgl. Sprachen und Impfschema) erfüllen. <br></br> Der Impfnachweis kann auf unterschiedliche Weise dokumentiert sein, eine Form ist das digitale <a href='#glossary_vaccination_certificate'>Impfzertifikat</a>."
                 },
                 {
                     "term": "Impfzertifikat",
                     "anchor": "vaccination_certificate",
-                    "description": "Das Impfzertifikat ist ein <a href='#glossary_digital_certificate'>digitales Zertifikat</a>, das einer Person ausgestellt wird, die eine Impfung mit einem in der EU zugelassenen Impfstoff wahrgenommen haben. Jedes Impfzertifikat dokumentiert genau einen Impfvorgang (eine Dosis). <br/><br/> Das Impfzertifikat stellt eine digitale Form für einen <a href='#glossary_vaccination_proof'>Impfnachweis</a> dar."
+                    "description": "Bestätigung über eine durchgeführte Impfung. Das digitale Impfzertifikat wird durch das Scannen von QR-Codes auf den vom jeweiligen medizinischen Personal erhaltenen Dokumenten erstellt. Jedes Impfzertifikat dokumentiert genau einen Impfvorgang (eine Dosis).<br></br>Das Impfzertifikat stellt eine digitale Form für einen <a href='#glossary_proof_of_vaccination'>Impfnachweis</a> dar."
+                },
+                {
+                    "term": "Impfstatus",
+                    "anchor": "vaccination_status",
+                    "description": "Zum Impfstatus gehören Informationen wie die Anzahl an erfolgten Impfungen, der verwendete Impfstoff und die Gültigkeit der Impfung."
+                },
+                {
+                    "term": "iOS",
+                    "anchor": "ios",
+                    "description": "iOS ist ein von Apple entwickeltes mobiles Betriebssystem für das iPhone, iPod touch und ehemalig auch iPad. Neben <a href='#glossary_android'>Android</a> ist es eines der beiden Betriebssysteme, die die Corona-Warn-App unterstützt."

```

Comment:
unterstützen

Reply:
Reworded the entire last sentence to prevent such unclarities, thanks for noticing!